webapp/app.py

```diff
@@ -33,17 +27,16 @@
 
 
 def create_app(testing=False):
-    app = flask.Flask(
-        __name__, template_folder="../templates", static_folder="../static"
+    app = FlaskBase(
+        __name__,
+        "snapcraft.io",
+        template_folder="../templates",
+        static_folder="../static",
+        template_404="404.html",
     )
-
     app.config.from_object("webapp.config")
     app.testing = testing
 
-    app.wsgi_app = ProxyFix(app.wsgi_app)
-    if app.debug:
-        app.wsgi_app = DebuggedApplication(app.wsgi_app)
-
     app.url_map.strict_slashes = False
     app.url_map.converters["regex"] = helpers.RegexConverter
```

Comment:
These 2 lines are also included in flask-base:

https://github.com/canonical-web-and-design/canonicalwebteam.flask-base/blob/master/canonicalwebteam/flask_base/app.py#L53

Reply:
I get an error related with this, Werkzeug is [used in Flask-base here](https://github.com/canonical-web-and-design/canonicalwebteam.flask-base/blob/master/canonicalwebteam/flask_base/converters.py#L1) but it is not a dependency in the library:
```
  File "/home/fran/Workspace/snapcraft.io/webapp/app.py", line 9, in <module>
    from canonicalwebteam.flask_base.app import FlaskBase
  File "/usr/local/lib/python3.6/dist-packages/canonicalwebteam/flask_base/app.py", line 7, in <module>
    from werkzeug.middleware.proxy_fix import ProxyFix
ModuleNotFoundError: No module named 'werkzeug.middleware'
```

Reply:
Solved!